XenAdmin/Wizards/ImportWizard/ImportSelectHostPage.cs

```diff
@@ -208,6 +234,69 @@ private bool CheckRightGpuExists()
             return true;
         }
 
+        public bool ApplianceCanBeStarted => _ovfMaxVCpusCount == null ||
+                                             GetPhysicalCpus(SelectedTarget ?? SelectedTargetPool) < 0 ||
+                                             GetPhysicalCpus(SelectedTarget ?? SelectedTargetPool) >= _ovfMaxVCpusCount;
+
+        private bool CheckDestinationHasEnoughPhysicalCpus(out string warningMessage)
+        {
+            warningMessage = string.Empty;
+            
+            if (ApplianceCanBeStarted)
+                return true;
+
+            var selectedTarget = SelectedTarget ?? SelectedTargetPool;
+            var physicalCpusCount = GetPhysicalCpus(selectedTarget);
+
+            if (physicalCpusCount < 0 || physicalCpusCount >= _ovfMaxVCpusCount)
+                return true;
+
+            if (selectedTarget is Pool)

```

Comment:
I'm also wondering whether the user might wonder why we warn about inadequate CPUs but not memory (which we do on the Properties or NewVM). It would be nice if the behaviour were uniform.

Reply:
About the second point, I'm a bit concerned about DMC implementation around this. It would probably be best if we logged this as an improvement under a separate ticket so we can test it effectively.